Core_OverlayMods/Clothes/KoiClothesOverlayController.cs

```diff
@@ -586,6 +637,14 @@ private static void SetOverlayExtData(Dictionary<CoordinateType, Dictionary<stri
             else
                 data.data.Remove(OverlayDataKey);
         }
+        private static void SetTextureSizeOverrideExtData(Dictionary<CoordinateType, Dictionary<string, int>> allTextureSizeOverrides, PluginData data)
+        {
+            
+            if (allTextureSizeOverrides.Count > 0)
```

Comment:
```suggestion
            if (allTextureSizeOverrides.Count > 0)
```

Reply:
Also, shouldn't the overrides be cleaned up before saving to remove all of the default values? If there are default values saved then on card load it goes through the whole load process even if there are no actual changes to apply. It could be added to CleanupTextureList.